lib/api/controllers/adminController.js

```diff
@@ -42,8 +43,7 @@ function AdminController (kuzzle) {
     return kuzzle.pluginsManager.trigger('data:beforeGetMapping', requestObject)
       .then(newRequestObject => {
         modifiedRequestObject = newRequestObject;
-
-        return kuzzle.services.list.readEngine.getMapping(modifiedRequestObject);
+        return engine.getMapping(modifiedRequestObject);
```

Comment:
why replacing `readEngine` by `writeEngine` here? (I know we are discussing about merging them, but actually, it's not done yet... or, if we replace it here, why not replacing it anywhere in this controller?)